src/layer/index.ts

```diff
@@ -705,11 +705,11 @@ export class ManagedUserLayer extends RefCounted {
   set name(value: string) {
     if (value !== this.name_) {
       this.name_ = value;
-      this.layerChanged.dispatch();
```

Comment:
@seankmartin I have a doubt on this line, it doesn't make sense to me, and I'm not sure it's a mistake and something I removed by error or if it's necessary. What do you think?

Reply:
@aranega I'm thinking we should put this line back. For the most post the different is not very noticable because the ` changeLayerName` function is often called instead of directly setting the name. And `changeLayerName` dispatches the same `layerChanged` signal.

There is however a duplicate signal for the change from `changeLayerName` because of this. So what we could do is to remove the dispatch in `changeLayerName` and keep it here in `set name` instead. Because that way `changeLayerName` still causes a dispatch (but only 1 now instead of 2). But a dispatch will also happen if the name is directly set for some reason outside of `changeLayerName`.